docs/tutorial.md

```diff
@@ -220,45 +147,33 @@ have a profile image. We start by defining a factory for profile image model
 ```javascript
 factory.define('ProfileImage', ProfileImage, {
   id: factory.sequence('ProfileImage.id'),
-  imageUrl: http://lorempixel.com/200/200
+  imageUrl: 'http://lorempixel.com/200/200'
 });
 ```
 
 To associate a profile image with factory generated models, we can simply do:
 
 ```javascript
 factory.define('User', User, {
-  email: factory.sequence('user.email', (n) => `dummy_user_${n}@my_domain.com`),
-  password: factory.chance('word', { syllables: 4 }),
-  name: factory.chance('name', { middle: true }),
-  about: factory.chance('paragraph', { sentences: 2 }),
-  password_expiry: () => methodToAddMonths(new Date(), 1),
-  favorite_cn_joke: () => { 
-    fetch('http://api.icndb.com/jokes/random')
-      .then(response => response.value.joke)
-  },
-  profile_image: factory.assoc('ProfileImage', 'id')
+  ...
+  profileImage: factory.assoc('ProfileImage', 'id')
 });
 ```
 
-`factory-girl` will now create a `ProfileImage` instance and place its `id` 
-attribute in the `profile_image` attribute of the created `User` instance.
+`factory-girl` will now create a `ProfileImage` instance and place its `id` attribute in
+the `profileImage` attribute of the created `User` instance.
 
-What if you want the `ProfileImage` instance itself to be assigned to 
-`profile_image`? Just don't pass `'id'` and the `ProfileImage` instance itself
-will be assigned to the `profile_image` attribute.
-
-Note that `factory.assoc` will persist the model instance to DB. In case you
- don't want the model to be persisted, you can instead use `factory.assocBuild`,
- that would just build the model instance and not persist it to DB.
- 
-If you are looking to just embed the model attributes (not the instance of the 
-model), use `factory.assocAttrs` in place of `factory.assoc`.
+What if you want the `ProfileImage` instance itself to be assigned to `profileImage`? Just
+don't pass `'id'` and the `ProfileImage` instance itself will be assigned to the
+`profileImage` attribute.
 
+> Note that `factory.assoc` will persist the model instance to DB. In case you don't want
+the model to be persisted, use `factory.assocAttrs` which just builds the model instance
```

Comment:
`assocAttrs` will just set the model attributes, probably `model instance` might be misleading.